src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -74,10 +74,11 @@ protected function doWrite($message, $newline)
             $message .= PHP_EOL;
         }
 
-        if (false === @fwrite($this->stream, $message)) {
+        if (!is_resource($this->stream) || !is_string($message)) {
```

Comment:
looking at the constructor, this cannot happen, so let's remove the block?

Reply:
Done. Should i keep at least !is_string($message) ? (can be totally removed in 5.0 thanks to typehint, but not in 3.4 and 4.4)

Reply:
let's add `$message = (string) $message;` at the very beginning of the method then, that'll be good.

Reply:
Done.